storage.conf

```diff
@@ -70,6 +70,11 @@ additionalimagestores = [
 #     Tells containers/storage where an ostree repository exists that might have
 #     previously pulled content which can be used when attempting to avoid
 #     pulling content from the container registry
+#   * convert_images = "false" | "true"
+#     If set to true, containers/storage will convert images to a
+#     format compatible with partial pulls in order to take advantage
+#     of local deduplication and hardlinking.  It is an expensive
```

Comment:
I realize it's a cut/paste from old stuff, but if you could fix it here and above, it would be appreciated.

```suggestion
#     of local deduplication and hard linking.  It is an expensive
```